console/package.json

```diff
@@ -32,20 +32,20 @@
 		"dist/*"
 	],
 	"dependencies": {
-		"@enact/agate": "enactjs/agate#1.0.0-alpha.3",
-		"@enact/core": "^3.0.0",
-		"@enact/i18n": "^3.0.0",
-		"@enact/spotlight": "^3.0.0",
-		"@enact/ui": "^3.0.0",
+		"@enact/agate": "enactjs/agate#2.0.0-alpha.3",
+		"@enact/core": "^4.0.2",
+		"@enact/i18n": "^4.0.2",
+		"@enact/spotlight": "^4.0.2",
+		"@enact/ui": "^4.0.2",
```

Comment:
```suggestion
		"@enact/agate": "^2.0.0-beta.1",
		"@enact/core": "^4.0.3",
		"@enact/i18n": "^4.0.3",
		"@enact/spotlight": "^4.0.3",
		"@enact/ui": "^4.0.3",
```

Reply:
Same here. "@enact/agate": "^2.0.0-beta.2" instead of "@enact/agate": "^2.0.0-beta.1" ?